addons/project_todo/views/project_task_views.xml

```diff
@@ -81,41 +81,45 @@
-                    <div class="py-1 px-2 border rounded bg-view">
-                        <field name="user_ids"
-                               widget="many2many_avatar_user"
-                               required="1"
-                               options="{'no_quick_create': True}"
-                               placeholder="Assignees"/>
-                    </div>
                     <field name="personal_stage_type_id"
                            domain="[('user_id','=',uid)]"
                            widget="statusbar"
                            options="{'clickable': '1', 'fold_field': 'fold'}"/>
                     <field name="active" invisible="1"/>
-                    <field name="state" invisible="1"/>
                 </header>
                 <sheet class="o_todo_form_sheet_bg">
                     <widget name="web_ribbon" class="todo_archived" title="Archived" bg_color="text-bg-danger" invisible="active"/>
-                    <field name="description" type="html" class="oe_description" default_focus="1" options="{'resizable': false, 'collaborative': true}"/>
+                    <div class="oe_title pe-0">
+                        <h1 class="d-flex justify-content-between align-items-center">
+                            <div class="d-flex w-100">
+                                <field name="priority" widget="priority_switch" class="me-3"/>
+                                <field name="name" options="{'line_breaks': False}" widget="text" class="o_task_name text-truncate w-md-75 w-100 pe-2" placeholder="To-do..."/>
+                            </div>
+                            <div class="d-flex justify-content-end o_state_container">
+                                <field name="state" widget="todo_done_checkmark" class="o_task_state_widget" />
+                            </div>
+                        </h1>
+                    </div>
+                    <group>
+                        <group>
+                            <field name="user_ids"
+                                widget="many2many_avatar_user"
+                                required="1"
+                                options="{'no_quick_create': True}"
+                                placeholder="Assignees"/>
+                        </group>
+                        <group>
+                            <field name="tag_ids"
+                                widget="many2many_tags"
+                                options="{'color_field': 'color', 'no_create_edit': True}"
+                                class="me-2"/>
+                        </group>
+                    </group>
+                    <field name="description" type="html" class="oe_description" default_focus="1" options="{'resizable': false, 'collaborative': true}"
+                        placeholder="Type Here..."/>
                 </sheet>
-                <chatter/>
+                <widget name="todo_chatter_panel" groups="base.group_user"/>
```

Comment:
the `groups` attribute does not seem needed since that view is only available for internal users, do I miss something? 🧐

Reply:
Ok, I removed the attribute.